README.md

```diff
@@ -235,6 +253,93 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Optional Arguments
+
+* -c (Config file)
+
+Use the config file specified.
+
+The default locations that will be checked are ```/etc/pump.io.json``` and
+```~/.pump.io.json```, use this option to override it. The default config
+values are stored in lib/defaults.js.
+
+Example :
+
+    pump -c pump.io.json
+
+### Environment
+
+Set the env variable ```NODE_DEBUG``` to enable debugging.
+
+Example :
+
+    export NODE_DEBUG=dev,all,net,http,fs,tls,module,timers
+
+See http://www.juliengilli.com/2013/05/26/Using-Node.js-NODE_DEBUG-for-fun-and-profit/
+
+### Debugging
+
+Debugging can be done via the Node Inspector https://github.com/node-inspector/node-inspector
+
+    node-debug pump
+
+### Arguments
+
+Execute with the config file on the command line using -c
+
+The default locations are ```/etc/pump.io.json``` and ```~/.pump.io.json```.
+
+    ./bin/pump -c pump.io.json
+
+## Using the command line tools
+
+### pump-register-app
+
+First use this tool to create the credentials file
+
+    ./bin/pump-register-app  -t app-name
+
+This will create the file ```~/.pump.d/*server*.json``` that contains
+
+    {
+    "client_id":"XXXX",
+    "client_secret":"YYYYY",
+    "expires_at":0
+    }
+
+It will also add an entry into the database :
+
+
+You can dump the database :
```

Comment:
Wiki is fine with me. We can link to the wiki from the readme.

On Mon, Oct 19, 2015 at 7:45 PM, Alex Jordan <notifications@github.com>
wrote:

> In README.md
> <https://github.com/e14n/pump.io/pull/1114#discussion_r42440428>:
>
> > +First use this tool to create the credentials file
> > +
> > +    ./bin/pump-register-app  -t app-name
> > +
> > +This will create the file ```~/.pump.d/*server*.json``` that contains
> > +
> > +    {
> > +    "client_id":"XXXX",
> > +    "client_secret":"YYYYY",
> > +    "expires_at":0
> > +    }
> > +
> > +It will also add an entry into the database :
> > +
> > +
> > +You can dump the database :
>
> I'm fine with documenting how to use memcached, but it doesn't seem like
> it belongs in the official, in-tree documentation. It would be awesome on
> the wiki, though.
> Unless you disagree?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e14n/pump.io/pull/1114/files#r42440428>.
>



-- 
James Michael DuPont
Kansas Linux Fest http://kansaslinuxfest.us
Free/Libre Open Source and Open Knowledge Association of Kansas
http://openkansas.us
Member of Free Libre Open Source Software Kosova http://www.flossk.org
Saving Wikipedia(tm) articles from deletion http://SpeedyDeletion.wikia.com